MobSF/utils.py

```diff
@@ -327,3 +327,16 @@ def isDirExists(dir_path):
 
 def genRandom():
     return ''.join([random.SystemRandom().choice('abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)') for i in range(50)])
+
+
+def zipdir(path, zip_file):
+    """Zip a directory."""
+    try:
+        print "[INFO] Zipping"
+        # pylint: disable=unused-variable
```

Comment:
Why don't you `_` to denote the unused variable?
I am just curious, pylint won't consider that?